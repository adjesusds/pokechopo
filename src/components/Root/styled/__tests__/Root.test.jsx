// External Packages
import React from 'react'
import { create } from 'react-test-renderer'

// Internal Packages
import StyledRoot from '../Root'

describe('<StyledRoot />', () => {
  it('should style the component on valid props', () => {
    const styledComponent = create(<StyledRoot />)

    expect(styledComponent).toMatchSnapshot()
  })
})
