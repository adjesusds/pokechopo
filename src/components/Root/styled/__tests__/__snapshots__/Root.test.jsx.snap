// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledRoot /> should style the component on valid props 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;
